src/main/java/com/google/api/gax/grpc/Clock.java

```diff
@@ -37,39 +37,15 @@
  * for testing.
  */
 interface Clock {
```

Comment:
Add a note that we should get rid of this when the earliest version of Java we support is Java 8 (which has its own Clock class: https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html )